assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -141,9 +141,9 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
     isTextElement && selectedElement.content.length && isResizingFromCorner;
 
   const { isMedia } = getDefinitionForType(selectedElement.type);
-  const actionsEnabled =
-    !selectedElement.isFill &&
-    selectedElement.id !== currentPage.backgroundElementId;
+  const isBackgroundElement =
+    selectedElement.id === currentPage.backgroundElementId;
```

Comment:
This reminds me... We should probably add a `isBackground` property (maintained by the reducer) in the story provider - we often need this check and this way of doing it is rather annoying. And the reducer has full control of what element is and isn't background and already has a method for filtering out disallowed properties for new and updated elements (so no element tries to sneak in this property).

Reply:
Yeah that would make sense to me!

Reply:
I think we already have `isBackground` property there :)

Reply:
Even better!